src/main/java/org/greenbuttonalliance/gbaresourceserver/customer/model/CustomerAccount.java

```diff
@@ -53,14 +53,14 @@ public class CustomerAccount extends Document {
 	@Column(name = "last_bill_amount")
 	private Long lastBillAmount;
 
-	@OneToOne(cascade = CascadeType.ALL)
+	@OneToOne(cascade = {CascadeType.PERSIST, CascadeType.MERGE})
```

Comment:
Changes in these annotations are just in case `Organisations` are reused for multiple `CustomerAccount`s in the future, deleting one `CustomerAccount` won't try to automatically delete the `Organisation` used by another `CustomerAccount` as well. I was probably being unnecessarily cautious with this consideration, but I don't think it hurts either. Same goes for similar changes in other files.